ludwig/benchmarking/reporting.py

```diff
@@ -33,3 +39,92 @@ def create_metrics_report(experiment_name: str) -> Tuple[Dict[str, Any], str]:
     merged_file_path = os.path.join(os.getcwd(), experiment_name, "metrics_report", "{}.json".format("full_report"))
     save_json(merged_file_path, full_report)
     return full_report, merged_file_path
+
```

Comment:
Is there any logic in the ludwig/benchmarking folder that gets replaced by the logic added here?

Reply:
Yes, will be added in a separate PR